package.json

```diff
@@ -33,6 +36,7 @@
     "@changesets/cli": "2.29.7",
     "@commercetools-frontend/babel-preset-mc-app": "24.8.0",
     "@commercetools-frontend/eslint-config-mc-app": "24.8.0",
+    "@commercetools-frontend/mc-scripts": "^24.10.0",
```

Comment:
This has been added so we can generate the auth token required for the GraphQL types generation.